src/test/scala/se/nimsa/sbx/metadata/PropertiesDAOTest.scala

```diff
@@ -800,8 +806,37 @@ class PropertiesDAOTest extends AsyncFlatSpec with Matchers with BeforeAndAfterA
       st1.size should be(2)
       st2.size should be(2)
       st3.size should be(2)
-      st4.size should be(1)
-      st5.size should be(0)
+      st4.size should be(2)
+      st5.size should be(2)
+    }
+  }
+
+  it should "update a series tag" in {
+    for {
+      (_, (_, _), (dbSeries1, dbSeries2, dbSeries3, _), (_, _, _, _, _, _, _, _)) <- insertMetaDataAndProperties()
+      seriesTags1 <- propertiesDao.listSeriesTags
+      maybeUpdatedTag <- propertiesDao.updateSeriesTag(SeriesTag(seriesTags1.head.id, "UpdatedTag1"))
+      seriesTags2 <- propertiesDao.listSeriesTags
+    } yield {
+      seriesTags1.size should be(2)
+      seriesTags1.map(_.name) should be(List("Tag1", "Tag2"))
+      maybeUpdatedTag.isDefined shouldBe true
+      seriesTags1.filter(_.name == "Tag1").head.id shouldEqual maybeUpdatedTag.get.id
+      seriesTags2.map(_.name).exists(_ == "UpdatedTag1") shouldBe true
+    }
+  }
+
+  it should "delete a series tag" in {
+    for {
+      (_, (_, _), (dbSeries1, dbSeries2, dbSeries3, _), (_, _, _, _, _, _, _, _)) <- insertMetaDataAndProperties()
+      seriesTags1 <- propertiesDao.listSeriesTags
+      _ <- propertiesDao.deleteSeriesTag(seriesTags1.head.id)
+      seriesTags2 <- propertiesDao.listSeriesTags
+    } yield {
+      seriesTags1.size should be(2)
+      seriesTags1.map(_.name) should be(List("Tag1", "Tag2"))
+      seriesTags2.size should be(1)
+      seriesTags2.map(_.name).exists(_ == "Tag1") shouldBe false
     }
```

Comment:
Are we missing tests for create series tag or is this part of the other tests?

Reply:
"create" is tested implicit through insertMetaDataAndProperties() which is used in all tests